.jscsrc

```diff
@@ -0,0 +1,5 @@
+{
+  "preset": "airbnb",
+  "excludeFiles": ["!(app|lib)"],
```

Comment:
It seemed to be a bug with jshint's globbing - that glob says "exclude everything but `app` and `lib`", but jshint was including vendor for some reason too, so I added that rule there. jscs didn't have that issue.